src/Microsoft.HttpRepl.IntegrationTests/Commands/EchoCommandTests.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.IntegrationTests.Mocks;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.IntegrationTests.Commands
+{
+    public class EchoCommandTests : CommandTestsBase
```

Comment:
I am wondering if we should add tests for ExecuteAsync(..). Looks like it doesn't do much. But we could verify the EchoRequest value. I'll let you decide if its worth adding or not.